tidb-cloud/tidb-cloud-tls-connect-to-dedicated-tier.md

```diff
@@ -0,0 +1,321 @@
+
+3. In this dialog, you should setup three steps:
+   - Step 1：Create traffic filter
+   - Step 2：Download TiDB cluster CA
+   - Step 3：Connect with an SQL client
+
+4. In the Step1：Create traffic filter
+
+Specify the traffic allowed to connect the cluster. Traffic matching any filter is allowed, and all other traffic is denied. For detailed configuration, see: Configure your [IP Access List](/tidb-cloud/configure-ip-access-list.md)
+
+5. In the Step 2：Download TiDB cluster CA
+
+You should click **"**Download TiDB Cluster CA** to download it locally for client TLS configuration.Because of verifying the TiDB cluster CA can ensure the TLS connection is secure and reliable.
+
+6. In the Step 3：Connect with an SQL client
+
+Connect with a SQL client in the dialog, click the tab of your preferred connection method, and then refer to the connection string and sample code on the tab to connect to your cluster.
+
+   - Support Mysql CLI, MyCLI, JDBC, Python, Go, Node.js and other six ways to connect to your Dedicated Tier cluster
+   - Provide MacOS, Debian, RedHat, Alpine, OpenSUSE, Windows and other operating system configuration parameters
+
+The following examples show the connection string in MySQL CLI , MyCLI , JDBC, Python, Go and Node.js:
+
+<SimpleTab>
+<div label="MySQL CLI">
+
+MySQL CLI client attempts to establish TLS connection by default. When you connect to Dedicated Tier clusters, you should set `ssl-mode` and `ssl-ca`.
+
+```shell
+mysql --connect-timeout 15 --ssl-mode=VERIFY_IDENTITY --ssl-ca=ca.pem --tls-version="TLSv1.2" -u root -h tidb.eqlfbdgthh8.clusters.staging.tidb-cloud.com -P 4000 -D test -p
+```
+
+Parameter description：
+
+- With `--ssl-mode=VERIFY_IDENTITY`, MySQL CLI client forces to enable TLS and validate TiDB Dedicated Tier clusters.
+- Use `--ssl-ca=<CA_root_path>` to set the CA root path on your system.
+- Use `--tls-version=TLSv1.2,TLSv1.3` to restrict the versions of TLS protocol.
+
+</div>
+
+<div label="MyCLI">
+
+[MyCLI](https://www.mycli.net/) automatically enables TLS when using TLS related parameters. When you connect to TiDB Dedicated Tier clusters, you need to set `ssl-ca` and `ssl-verify-server-cert`.
+
+```shell
+mycli --ssl-ca=ca.pem --ssl-verify-server-cert -u root -h tidb.eqlfbdgthh8.clusters.staging.tidb-cloud.com -P 4000 -D test
+```
+
+Parameter description：
+
+- Use `--ssl-ca=<CA_root_path>` to set the CA root path on your system.
+- With `--ssl-verify-server-cert` to validate TiDB Dedicated Tier clusters.
+
+</div>
+
+<div label="JDBC">
+
+[MySQL Connector/J](https://dev.mysql.com/doc/connector-j/8.0/en/)'s TLS connection configurations are used here as an example.
+
+After downloaded TLS,if you want to import your downloaded TLS Certificate Template,Please refer to the order **keytool -importcert -alias TiDBCACert -file ca.pem -keystore <your_custom_truststore_path> -storepass <your_truststore_password>** 
```

Comment:
```suggestion
After downloaded cert, if you want to import your downloaded cert, please refer to the command **keytool -importcert -alias TiDBCACert -file ca.pem -keystore <your_custom_truststore_path> -storepass <your_truststore_password>** 
```

Reply:
updated but improve this sentence ，thanks